src/state/user/reducer.ts

```diff
@@ -55,6 +57,8 @@ function pairKey(token0Address: string, token1Address: string) {
 }
 
 export const initialState: UserState = {
+  // TODO(lynnshaoyu): change buyFiatFlowCompleted initial state to false after finishing redux migration.
```

Comment:
so you won't be able to change the initial state here later because some users will have already had `buyFiatFlowCompleted: undefined` as their persisted redux state

Reply:
oooh I see gotcha, ok that works fine with our use case